exporter/tanzuobservabilityexporter/transformer_test.go

```diff
@@ -401,41 +401,26 @@ func spanWithStatus(statusCode ptrace.StatusCode, message string) ptrace.Span {
 	return span
 }
 
-func TestAttributesToTagsForMetrics(t *testing.T) {
-	// 1. Empty sourceKey: does not change resulting wfTags
```

Comment:
Since this test had to be removed because the method was renamed and changed, would it make sense to add a new test for the ``` pointAndResAttrsToTagsAndFixSource``` method?

Reply:
Added code coverage.